include/gpac/isomedia.h

```diff
@@ -3204,6 +3204,14 @@ GF_VVCConfig *gf_isom_vvc_config_get(GF_ISOFile *isom_file, u32 trackNumber, u32
 */
 GF_AV1Config *gf_isom_av1_config_get(GF_ISOFile *isom_file, u32 trackNumber, u32 sampleDescriptionIndex);
 
+/*! gets AV1 scalable layer byte offsets for a1lx box
+\param isom_file the target ISO file
+\param trackNumber the target track
+\param layer_size returned 4 layer sizes
+\return none
+*/
+void gf_isom_av1_layer_size_get(GF_ISOFile *file, GF_ISOTrackID trackID, u32 layer_size[4]);
```

Comment:
could you make this function return GF_Err, BAD_PARAM if wrong params or not AV1 track, ISOM_INVALID_FILE if no av1 config ?